File: 10-9/InventoryManagement/main/java/com/example/registerloginex/SplashscreenActivity.java
package com.example.registerloginex;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import com.example.registerloginex.Activity.SigninActivity;

public class SplashscreenActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splashscreen);

        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {

                Intent i =new Intent(SplashscreenActivity.this, SigninActivity.class);
                startActivity(i);

            }
        },3000);
    }
}

File: 12-11/Project/Activity/SplashScreenActivity.kt
package com.example.e_greetingsheet.Activity

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler
import com.example.e_greetingsheet.R

class SplashScreenActivity : AppCompatActivity()
{
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash_screen)

        Handler().postDelayed(Runnable {

                var i =Intent(this,LoginActivity::class.java)
                startActivity(i)

        },3000)
    }
}